### DISCUSSION TITLE
Can FDS be used only for wind flow simulation?

### DISCUSSION THREAD

**Discussion by zhang-zhen-project** [firemodels/fds#12698] [Announcements]
Dear author,：
       We all know that FDS dominates the field of building fire. Can FDS only simulate the scene of ventilation without fire source, such as mine ventilation, and is it theoretically different from traditional ventilation simulation software such as Flunet? 
       Thank you for your answer!

**Comment by rmcdermo** [comment]
Yes, FDS can be used for local wind, pressure coefficients, and dispersion.  See examples in the manuals.

There are lots of ways in which FDS is different from Fluent.  But the key thing to note is that FDS is purely an LES code, whereas Fluent is a RANS code, which can be run as unsteady RANS.

The second notable difference is that Fluent is an unstructured code, whereas FDS is by default purely Cartesian, which can have important consequences if you need the details of complex geometry.  FDS has a beta feature for complex geometry that uses "cutcells", which you are welcome to test, but it is not recommended yet beyond testing and research use.
